README.md

```diff
@@ -69,6 +69,40 @@ Connector.getDAL(datasetId).
   });
 ```
 
+#### listen
+
+The Data Access Layer object for a collection has a `listen` function that ensures that the CRUDL topics for the data set are subscribed.
+
+```javascript
+
+/**
+* This function has access to the mongoose Model associated with the data set
+* through the "model" property
+*
+* @returns {Promise}
+*/
+function customListFunction() {
+  return this.model.list();
+}
+
+
+var datasetId = "workorders";
```

Comment:
I would do this as separate topic like : Overriding defaul CRUD methods and describe api there. 
It took me a while to understand what we documenting here.